polyglot/piranha/src/models/rule_store.rs

```diff
@@ -14,7 +14,7 @@ Copyright (c) 2022 Uber Technologies, Inc.
 use std::collections::HashMap;
 
 use colored::Colorize;
-use log::info;
+use log::{info};
```

Comment:
Needed? I think `log::info` is the right syntax if we are just importing that, no?

Reply:
YEs. Addressed by cargo format. 

I think I ll add a new Github action that runs cargo format on each PR, as a commit. Idk